### PR TITLE
Upgrade Steal to 1.x

### PR DESCRIPTION
**Pull request by Bajix**

### CODE REVIEW

**Comment by matthewp** [comment]
Hm, we need to figure out the test situation don't we.

**Comment by Bajix** [comment]
Yea testing this has been quite awkward

File: src/template.txt [hunk 1]
```diff
@@ -158,7 +158,7 @@ define(<%= imports %>, function(<%= args %>){
 		<%= ases %>
 	};
 	var isNW = (function(){
-		try{var nr = System._nodeRequire; return nr && nr('nw.gui') !== 'undefined';}catch(e){return false;}
+		try{var nr = steal._nodeRequire; return nr && nr('nw.gui') !== 'undefined';}catch(e){return false;}
```
Comment (Bajix) [inline src/template.txt]: @matthewp wasn't sure if this change was required for a Steal upgrade. Is this change potentially problematic?

**Comment by Bajix** [comment]
I can't make these tests fail locally

File: src/template.txt (hunk 1 above)
Comment (matthewp) [inline src/template.txt]: Yeah this shouldn't be changed. That's still System.

**Comment by matthewp** [comment]
I can probably take a look tomorrow.

**Comment by matthewp** [comment]
Taking a look at this now.

**Comment by matthewp** [comment]
Closing in favor of https://github.com/donejs/autorender/pull/36, which should pass soon.

**Pull request closed without merging**